### PR TITLE
Changed consume_drug from "Consume" to "Take" + style

### PR DESCRIPTION
**Pull request by Faalagorn**
Changed the flag's `consume_drug` name from "Consume" to "Take", since most of the items with this flags are pills that even state that you *take* then in activation message. There are a few odd items that still don't fit this message: items you smoke or inject and… tipping a fedora, though I think it's still better than "consuming" it (no, your fedora luckily is not being consumed upon activation) – these few items will probably need a separate treatment.

Also while at it, fixed some minor inconsistencies in few other action names.

### CODE REVIEW

**Comment by DracoGriffin** [comment]
I'm not sure about "take", as that ends up looking like this in-game:

Old: `You consume oxycodone.`
to
New: `You take oxycodone.`

Which can be confusing (translators would need to chime in here, might not be a big deal), as where are you taking it from? Consume, even if a bit obtuse, still conveys the same idea, where "take" is a bit more slang.

Although to be fair, even doctors and such will tell you "Make sure to **take** your pills at the right time or...".

So just need to consider for ESL players playing in English instead of a translated version.

**Comment by Faalagorn** [comment]
Are you sure it appears in anything than the activate menu? Currently (without this patch), when I press `a` I have an option to "Consume" aspirin, but the message still says "You take some aspirin" in log.

Also, for oxycodone currently there's `"activation_message" : "You take some oxycodone.",` in [med.json](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/json/items/comestibles/med.json#L853), not consume, so I this the this message should appear (with or without the patch)?

**Comment by cainiaowu** [comment]
I am against this change, it will merge with other "take" usage and make translator life miserable.

**Comment by cainiaowu** [comment]
The problem is that some injectable drugs is also using "consume_drug" actor, currently the "consume" is translated vaguely as "to use something in such a way that it cannot be reused or recovered afterward" and "take" is translated as "eat/drink".
The whole activate menu is kind of messy due to various contributors adding their use actions in it, it would be great if someone can clear it up a bit, which should split it into more groups to describe different actions rather than merging them.

**Comment by DracoGriffin** [comment]
>The whole activate menu is kind of messy due to various contributors adding their use actions in it, it would be great if someone can clear it up a bit, which should split it into more groups to describe different actions rather than merging them.

Starting a separate Issue with these outlined (especially from translator point of view) would be incredibly useful and potentially resolved much sooner.

**Comment by Faalagorn** [comment]
I mentioned that on Discord, but quoting myself here for visibility:

> Added styling to my old unmerged PR I made from GitHub, hopefully it's still all right https://github.com/CleverRaven/Cataclysm-DDA/pull/24648 I didn't try to rebase, but the styled file didn't even change since (last change was over 4 days ago)

> my old PR got `AppVeyor was unable to build non-mergeable pull request` now, but I'm if I can deal with it :/ Let me know if there's something I need to be done, but preferrably in comments under each PR as I'm gonna be leaving soon

File: data/json/item_actions.json [hunk 1]
```diff
@@ -2,12 +2,12 @@
   {
     "type": "item_action",
     "id": "repair_fabric",
-    "name": "Repair cotton/leather/wool/Nomex/fur"
+    "name": "Repair cotton/fur/leather/wool/Nomex"
```
Comment (Night-Pryanik) [inline data/json/item_actions.json]: That's just another meaningless work for translators.

File: data/json/item_actions.json [hunk 2]
```diff
@@ -2,12 +2,12 @@
...
   },
   {
     "type": "item_action",
     "id": "repair_metal",
-    "name": "Repair metal/plastic/kevlar"
+    "name": "Repair metal/plastic/Kevlar"
```
Comment (Night-Pryanik) [inline data/json/item_actions.json]: Kevlar was lowercased once, no need to uppercase it again.

File: data/json/item_actions.json [hunk 3]
```diff
@@ -37,7 +37,7 @@
   {
     "type": "item_action",
     "id": "CROWBAR",
-    "name": "Pry crate, window or door"
+    "name": "Pry crate/door/window"
```
Comment (Night-Pryanik) [inline data/json/item_actions.json]: That's just another meaningless work for translators.

File: data/json/item_actions.json [hunk 4]
```diff
@@ -62,7 +62,7 @@
   {
     "type": "item_action",
     "id": "BOLTCUTTERS",
-    "name": "Cut bolts or wires"
+    "name": "Cut bolts/wires"
```
Comment (Night-Pryanik) [inline data/json/item_actions.json]: That's just another meaningless work for translators.

File: data/json/item_actions.json [hunk 5]
```diff
@@ -72,7 +72,7 @@
   {
     "type": "item_action",
     "id": "MISC_REPAIR",
-    "name": "Repair wood/paper/bone/chitin"
+    "name": "Repair bone/chitin/paper/wood"
```
Comment (Night-Pryanik) [inline data/json/item_actions.json]: That's just another meaningless work for translators.

File: data/json/item_actions.json (hunk 1 above)
Comment (Faalagorn) [reply data/json/item_actions.json]: I know now :unamused:. I'd love to have it changed for English if only for the sake of consistency, but I hate the fact that every change in English file removes the translations from another languages, especially in cases such as these where each language. I can handle Polish as I am know registered in Transifex, but to help with other languages I'd have to join all the languages plus I'd have to remember to copy all respective strings beforehand (doable but definitely PITA).

I'll write an issue report, so maybe something could be done to at least mitigate those in the future, and will address #24761 first, as there's way more strings to be concerned here.

**Comment by ZhilkinSerg** [comment]
Marking this PR as Stalled to free up PR queue. Ping me to reopen if neccessary.

**Pull request closed without merging**